### ISSUE TITLE
SlevomatCodingStandard.Namespaces.UnusedUses false positive for annotations 

### ISSUE THREAD

**Issue by oldy777** [slevomat/coding-standard#1309]
It seems that rule doesn't recognize class usage in annotations
```
"name": "slevomat/coding-standard",
"version": "7.0.16",
```

```
use App\UtilBundle\Validator\Constraint as CustomAssert;
use Symfony\Component\Validator\Constraints as Assert;

final class Request
{
    /**
     * @Assert\NotBlank(message=CustomAssert\AbstractConstraint::VIOLATION_IS_REQUIRED) <----
     * @Assert\Type("int", message=Assert\Type::INVALID_TYPE_ERROR)
     * @Assert\Positive(message=Assert\Positive::TOO_LOW_ERROR)
     */
    private int $spaceId;

    /**
     * @Assert\NotBlank(message=CustomAssert\AbstractConstraint::VIOLATION_IS_REQUIRED) <----
     * @Assert\Type("int", message=Assert\Type::INVALID_TYPE_ERROR)
     * @Assert\Positive(message=Assert\Positive::TOO_LOW_ERROR)
     */
    private int $clientId;
```
```
------------------------------------------------------------------------------------------------------------------------------------------------------------------
FOUND 1 ERROR AFFECTING 1 LINE
------------------------------------------------------------------------------------------------------------------------------------------------------------------
 7 | ERROR | [x] Type App\UtilBundle\Validator\Constraint (as CustomAssert) is not used in this file.
   |       |     (SlevomatCodingStandard.Namespaces.UnusedUses.UnusedUse)
------------------------------------------------------------------------------------------------------------------------------------------------------------------
```

**Comment by kukulich** [comment]
Thank you for your report.

Fixed in https://github.com/slevomat/coding-standard/commit/d552bf37cee3ac0e733fac2a23ebe3d13b78acf3
